src/styl/_helpers/_cursor.styl

```
@@ -0,0 +1,13 @@
+// Cursor
+//
+// Change cursor in differents kind of specific situations
+// - **.cursor-zoom-in**: Change cursor to a magnifying glass to indicate to the user a zoom + .
+//
+// Markup: helpers-cursor.twig
+//
+// Styleguide: Helpers.Cursor
+
+.cursor
+    &-zoom
+        &-in
```

```suggestion
    &-zoomIn
```
No need to add another level here, this is just fine

---

ok i think is most easiest to complete class if one day we need to use the zoomOut. but it's ok for me too